src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -87,6 +109,9 @@ export const generateContent: TranslationFn = content => {
   const applicantApplyForConditionalOrderStarted = isApplicant2
     ? 'applicant2ApplyForConditionalOrderStarted'
     : 'applicant1ApplyForConditionalOrderStarted';
+  const isClarificationResponsesOrDocumentsUploaded =
```

Comment:
i dont think this is necessary as mentioned in the template.

Reply:
Removed isClarificationResponsesOrDocumentsUploaded.